OpenOversight/app/templates/submit_image.html

```diff
@@ -76,12 +76,20 @@ <h3>Drop images here to submit photos of officers:</h3>
             acceptedFiles: "image/png, image/jpeg, image/gif, image/jpg, image/webp",
             maxFiles: 50,
             headers: {
+                'Accept': 'application/json',
```

Comment:
This looks duplicates on the submit officer image page - is there any way these can be unified or do they differ enough to warrant separation?

Reply:
Good point, I've moved them to a new file